en/docs/services/service-concepts.md

```diff
@@ -8,39 +8,39 @@ A service in Choreo is a logical representation of a repeatable activity that ha
 ## Service Sample
 A service sample in Choreo is a prebuilt service that performs a specific activity. Choreo provides a comprehensive set of service samples for you to clone, edit, and try out, depending on your requirement. 
 
-## Connection
-A connection in Choreo allows you to connect to and interact with an external service through your application. You can use a connection to interact with generic protocol-based APIs and application-specific APIs.
+## API Call
+An API call in Choreo allows you to connect to and interact with an external service through your service. You can use an API call to interact with generic protocol-based APIs and service-specific APIs.
```

Comment:
Why we are changing the `application-specific API` to `service-specific API`? Application specific API refers to common 3rd party application-related APIs .

Reply:
Changed it back.